cupy_backends/cuda/libs/cublas.pyx

```diff
@@ -1004,6 +1022,59 @@ cpdef zgetrfBatched(intptr_t handle, int n, size_t Aarray, int lda,
     check_status(status)
 
 
+cpdef int sgetrsBatched(intptr_t handle, int trans, int n, int nrhs,
+                        size_t Aarray, int lda, size_t devIpiv,
+                        size_t Barray, int ldb, int batchSize):
+    cdef int info
+    _setStream(handle)
+    with nogil:
+        status = cublasSgetrsBatched(
+            <Handle>handle, <Operation>trans, n, nrhs,
+            <const float**>Aarray, lda, <const int*>devIpiv,
+            <float**>Barray, ldb, &info, batchSize)
+    check_status(status)
+    return info
```

Comment:
I think it's better to follow other cuBLAS routines (like `sgetrfBatched` etc) and pass in the info array instead of returning an int here. This return value is not used by your `batched_gesv` anyway.

Reply:
Actually I was wondering which was better. My first implementation followed other routine's convention, but changed it to current design as info is a host array with size 1 and it is easier to handle it with current design.
Anyway, I agree that current design is not consistent with other routines. So, I will change it to follow other routines.

Reply:
Oh oh I see what you're saying! In most routines `infoArray` is a device array (of size `batch_size`, but in this case it's a host array of size 1! It's very inconsistent API design...😞 

Another inconsistent design is that the batched version lives in cuBLAS but the non-batched version lives in cuSOLVER !@#$%...Otherwise we could just merge `cupy.cublas.batched_gesv` with `cupy.cusolver.gesv`...(Or should we do that anyways?)

Reply:
There's a historical reason for that: when we first added the Lapack-like routines to the CUDA library, there was no cuSOLVER yet..